stdlib/builtins.pyi

```diff
@@ -75,6 +75,7 @@ _TBE = TypeVar("_TBE", bound="BaseException")
 _R = TypeVar("_R")  # Return-type TypeVar
 _SupportsNextT = TypeVar("_SupportsNextT", bound=SupportsNext[Any], covariant=True)
 _SupportsAnextT = TypeVar("_SupportsAnextT", bound=SupportsAnext[Any], covariant=True)
+_B = TypeVar("_B", bool, int)
```

Comment:
In my experience `mypy` does not have ideal support for TypeVars with values (and bugs in this field are really hard to fix!). It might be the root cause in `pandas` case.